app/controllers/application_controller.rb

```diff
@@ -634,6 +615,10 @@ def build_ae_tree(type = :ae, name = :ae_tree)
     tree
   end
 
+  def filename_timestamp(basename, format = 'fname')
```

Comment:
You can include a helper in a controller with the `helper` method: http://api.rubyonrails.org/classes/ActionController/Helpers.html